core/src/main/java/org/apache/accumulo/core/fate/Fate.java

```diff
@@ -174,7 +174,8 @@ private void transitionToFailed(long tid, Exception e) {
       // as a warning. They're a normal, handled failure condition.
       if (e instanceof AcceptableException) {
         var tableOpEx = (AcceptableThriftTableOperationException) e;
-        log.debug(msg + " for {}({}) {}", tableOpEx.getTableName(), tableOpEx.getTableId(),
+        log.debug(msg + " for {}({}) op:{} exception-type:{} desc:{}", tableOpEx.getTableName(),
```

Comment:
Could avoid the string concat and pass
```suggestion
        log.debug("{} for {}({}) op:{} exception-type:{} desc:{}",msg,  tableOpEx.getTableName(),
```